trace_replay/block_cache_tracer.cc

```diff
@@ -0,0 +1,223 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#include "trace_replay/block_cache_tracer.h"
```

Comment:
Adjust the order of header inclusion.

Reply:
I thought the .cc file should put its .h file in the beginning, right?

Reply:
Yes you are right.